src/createServer.js

```diff
@@ -1,3 +1,33 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const { checkErrors } = require('./checkErrors');
+
+const HOST = 'http://localhost:5700';
```

Comment:
Are you sure that it always be localhost? better to use host from request instead